onnxruntime/core/providers/qnn/builder/qnn_backend_manager.h

```diff
@@ -28,11 +30,13 @@
   QnnBackendManager(std::string backend_path,
                     ProfilingLevel profiling_level,
                     uint32_t rpc_control_latency,
-                    HtpPerformanceMode htp_performance_mode)
-      : backend_path_(backend_path),
+                    HtpPerformanceMode htp_performance_mode,
+                    std::string qnn_saver_path = "")
+      : backend_path_(std::move(backend_path)),
```

Comment:
>std::move(backend_path [](start = 22, length = 22)

Since the backend_path_ and qnn_saver_path_ in QNNExecutionProvider is only used for QnnBackendManager, we can remove it from QNNExecutionProvider  class, define it as local variable. Here we can define it as QnnBackendManager(std::string&& backend_path, ...

Reply:
Updated.